docs/hosting/installation/server-setups/docker-compose.md

```diff
@@ -1,70 +1,128 @@
 ---
 #https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
 contentType: tutorial
+description: Install and run n8n using Docker Compose
 ---
 
 # Docker-Compose
 
-If you have already installed Docker and Docker-Compose, then you can start with step 4.
+If you have already installed Docker and Docker-Compose, then you can start with [step 3](#3-dns-setup).
 
 You can find Docker Compose configurations for various architectures in the [n8n-hosting repository](https://github.com/n8n-io/n8n-hosting).
 
 --8<-- "_snippets/self-hosting/warning.md"
 
 --8<-- "_snippets/self-hosting/installation/latest-next-version.md"
 
-### 1. Install Docker
+## 1. Install Docker and Docker Compose
 
-This can vary depending on the Linux distribution used. You can find detailed instructions in the [Docker documentation](https://docs.docker.com/engine/install/){:target=_blank .external-link}. The following example is for Ubuntu:
+How you install Docker and Docker Compose can vary depending on the Linux distribution you use. You can find detailed instructions in [Docker](https://docs.docker.com/engine/install/) and [Docker Compose](https://docs.docker.com/compose/install/) installation documentation. The following example is for Ubuntu:
```

Comment:
```suggestion
How you install Docker and Docker Compose can vary depending on the Linux distribution you use. You can find detailed instructions in both the [Docker](https://docs.docker.com/engine/install/) and [Docker Compose](https://docs.docker.com/compose/install/) installation documentation. The following example is for Ubuntu:
```

Reply:
Good catch! I think I removed some things from this sentence and forgot to read the whole thing again :smile: